notebooks/cybert/cybert_example_training.ipynb

```diff
@@ -44,6 +44,38 @@
     "import numpy as np"
```

Comment:
<p>looks good to me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/efajardo-nv/clx/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>